src/simulator.ts

```diff
@@ -99,7 +99,7 @@ export class Simulator {
     if (eventName === Constants.SimulatorLaunchEvent) {
       TelemetryClient.sendEvent(eventName, {
         Result: result ? "Success" : "Fail",
-        Error: result ? undefined : properties.error,
+        [Constants.errorProperties.Message]: result ? undefined : properties.error,
```

Comment:
Please also update property name in other events if you want to rename this property. For example, line 117 in this file.

Reply:
Good catch!